src/plugins/yy/yyplugin.cpp

```diff
@@ -52,6 +53,33 @@ static QString sanitizeName(QString name)
     return name.replace(regexp, QStringLiteral("_"));
 }
 
+/**
+ * Determine the resource identifier associated with the given filepath.
+ * This is achieved by looking for a *.yy meta file and, if found, returning
+ * its name without the extension. The function starts searching the current
+ * directory and up to two parent directories to account for a typical
+ * project directory structure.
+ * If no *.yy file can be found, returns the filename without its extension.
+ */
+static QString determineResourceId(const QString &filePath)
+{
+    constexpr unsigned int searchDepth = 3;
+    QFileInfo fi(filePath);
+    QDir dir(fi.canonicalPath());
```

Comment:
I wasn't aware that this is a valid assumption. I was under the impression that Tiled could use relative paths to keep a project portable. But regardless, not resolving symbolic links might actually be the more logical thing to do.

Reply:
Tiled writes relative paths when writing out files, but they are not kept as relative in memory. They need to be resolved to absolute on loading, because later on we'd no longer know what they should be relative to.

Reply:
Done.